### DISCUSSION TITLE
Close to Tray in Windows Client

### DISCUSSION THREAD

**Discussion by csch0en** [BearWare/TeamTalk5#2245] [Ideas]
Hi.
The windows client supports "minimize to tray" that's fine, but it will be also a good feature to implement "close to tray", for people who like to klick the "x" and keep the app running in background. Like me :-) Several other social media clients support "close to tray". 
BR.

**Comment by muamel1** [comment]
I believe there is an option for that.

**Reply by csch0en** [reply]
Have not found anything about "close to tray" options in the app or in the config file. 
Are you sure?

My workaround is to use a 3rd party app called "DesktopOK" developed by http://www.softwareok.de. It can sent most apps to tray when click the x with right mouse button.

**Reply by muamel1** [reply]
Take a look at the display settings. You'll find it there.
